core/src/main/kotlin/io/rover/campaigns/core/CoreAssembler.kt

```diff
@@ -465,6 +478,8 @@ class CoreAssembler @JvmOverloads constructor(
             // deschedule any prior rover sync jobs.
             WorkManager.getInstance().cancelAllWorkByTag("rover-sync")
         }
+
+        application.registerActivityLifecycleCallbacks(AutoScreenTracker())
```

Comment:
I'm not sure how I feel about enabling/disabling the auto tracking feature globally at the Manifest level.  The other features are largely enabled & disabled with boolean arguments to the assemblers.
 
`analyticsActivityAutoTracking: Boolean = true` as a parameter to CoreAssembler, perhaps?

Reply:
Yeah, I thought that doing all the config in the manifest would be easier to use, but after I bit of thought I think you're right

Reply:
But I also think autotracking should be off by default

Reply:
as discussed off-line, all agreed